src/main/java/org/isf/medicalstock/gui/MovStockMultipleDischarging.java

```diff
@@ -596,7 +591,7 @@ protected Lot chooseLot(List<Lot> lots, double qty) {
 	private boolean checkQuantityInLot(Lot lot, double qty) {
 		double lotQty = lot.getMainStoreQuantity();
 		if (qty > lotQty) {
-			MessageDialog.error(this, "angal.medicalstock.movementquantityisgreaterthanthequantityof.msg");
+			MessageDialog.error(this, "angal.medicalstock.movementquantityisgreaterthanthequantityof.msg", qty, lotQty);
```

Comment:
shouldn't messages that take arguments include ".fmt"?
```suggestion
			MessageDialog.error(this, "angal.medicalstock.movementquantityisgreaterthanthequantityof.fmt.msg", qty, lotQty);
```